.github/workflows/ci.yml

```diff
@@ -129,12 +141,11 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        compiler: [gcc]
         image: [windows-2019, windows-2022]
     runs-on: ${{ matrix.image }}
     env:
-      CC: ${{ matrix.compiler }}
-      SUFFIX: windows-${{ matrix.image }}-${{ matrix.compiler }}
+      CC: gcc
+      SUFFIX: windows-${{ matrix.image }}-gcc
```

Comment:
Do `file jq.exe` for windows build also? .. sorry can't comment further down, no diff

Reply:
Yes, added.